pkg/kubelet/kubelet.go

```diff
@@ -1943,6 +1943,11 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
+		// TODO(ndixita) move to status manager and set it equal to pod requests
+		pod.Status.AllocatedResources = pod.Spec.Resources.Requests
```

Comment:
for container resize, this is done here: https://github.com/kubernetes/kubernetes/blob/9630ab9581afbac9835d53f9e620a1240a1d2d91/pkg/kubelet/kubelet_pods.go#L2411

your todo to move this to status manager is throwing me off, was there a reason to do it there rather than in kubelet_pods.go like we do for container resize?